ui/raidboss/data/05-shb/raid/e2s.js

```diff
@@ -36,11 +36,11 @@
   triggers: [
     {
       id: 'E2S Spell In Waiting Gain',
-      regex: / 1A:\y{ObjectId}:Voidwalker gains the effect of Spell-In-Waiting/,
-      regexCn: / 1A:\y{ObjectId}:虚无行者 gains the effect of 延迟咏唱/,
-      regexDe: / 1A:\y{ObjectId}:Nichtswandler gains the effect of Verzögerung/,
-      regexFr: / 1A:\y{ObjectId}:Marcheuse Du Néant gains the effect of Déphasage Incantatoire/,
-      regexJa: / 1A:\y{ObjectId}:ヴォイドウォーカー gains the effect of ディレイスペル/,
+      regex: Regexes.gainsEffect({ effect: 'Spell-In-Waiting', capture: false }),
```

Comment:
These need `target: 'Voidwalker'` in them.

Reply:
done